### PR TITLE
KHADRI-3-BUG-1 Client 2 Requirement Based Code Changed

### PR DESCRIPTION
**Pull request by Babjanshake**

### CODE REVIEW

File: src/college-mapping.xml [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-		<property name="clgName" type="string" column="collegename"/>
+		<property name="clgRegNumber" type="string" />
+		<property name="clgName" type="string"/>
+		<property name="clgStrength" type="int"/>
+		<property name="clgAddress" type="string"/>
+		<property name="clgTown" type="string"/>
+		<property name="clgCity" type="string"/>
+		<property name="clgState" type="string"/>
+		<property name="clgAnnualReturns" type="double"/>
+		<property name="clgOfficeStaff" type="int"/>
```
Comment (khadri2021) [inline src/college-mapping.xml:15]: use camelcase

File: src/com/khadri/hibernate/client/Client2.java [hunk 2]
```diff
@@ -1,64 +1,55 @@
...
 
 import org.hibernate.Session;
-import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
-import org.hibernate.cfg.Configuration;
 
 import com.khadri.hibernate.entity.College;
+import com.khadri.hibernate.session.BasicOperationSession;
 
 public class Client2 {
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/Client2.java]: change Client2 to StoreCollege.java

File: src/com/khadri/hibernate/client/Client2.java [hunk 3]
```diff
@@ -1,64 +1,55 @@
...
+		System.out.println("Enter Town Of The College : ");
+		String clgTown = sc.next();
+		System.out.println("Enter City Of The College : ");
+		String clgCity = sc.next();
+		System.out.println("Enter State Of The College : ");
+		String clgState = sc.next();
+		System.out.println("Enter How Much Annual Return Of The College : ");
+		double clgAnnualReturns = sc.nextDouble();
+		System.out.println("Enter How Many Office Staff in The College : ");
+		int clgOfficeStaff = sc.nextInt();
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/Client2.java]: use camelcase

File: src/com/khadri/hibernate/client/Client2.java [hunk 4]
```diff
@@ -1,64 +1,55 @@
...
+		college.setClgTown(clgTown);
+		college.setClgCity(clgCity);
+		college.setClgState(clgState);
+		college.setClgAnnualReturns(clgAnnualReturns);
+		college.setClgOfficeStaff(clgOfficeStaff);
+		college.setClgTeachingStaff(clgTeachingStaff);
+		
+		session.save(college);
+		txn.commit();
+		System.out.println("Successfully Inserted.");
```
Comment (khadri2021) [inline src/com/khadri/hibernate/client/Client2.java]: Record Inserted Successfully

File: src/com/khadri/hibernate/entity/College.java [hunk 5]
```diff
@@ -2,7 +2,16 @@
...
 	private Integer clgId;
+	private String clgRegNumber;
 	private String clgName;
+	private Integer clgStrength;
+	private String clgAddress;
+	private String clgTown;
+	private String clgCity;
+	private String clgState;
+	private Double clgAnnualReturns;
+	private Integer clgOfficeStaff;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/entity/College.java:13]: use camel case

File: src/com/khadri/hibernate/entity/College.java [hunk 6]
```diff
@@ -20,9 +37,75 @@ public void setClgName(String clgName) {
...
+		this.clgTeachingStaff = clgTeachingStaff;
+	}
+
 	@Override
 	public String toString() {
-		return "College [clgId=" + clgId + ", clgName=" + clgName + "]";
+		return "College [clgId=" + clgId + ", clgRegNumber=" + clgRegNumber + ", clgName=" + clgName + ", clgStrength="
+				+ clgStrength + ", clgAddress=" + clgAddress + ", clgTown=" + clgTown + ", clgCity=" + clgCity
+				+ ", clgState=" + clgState + ", clgAnnualReturns=" + clgAnnualReturns + ", clgOfficeStaff="
+				+ clgOfficeStaff + ", clgTeachingStaff=" + clgTeachingStaff + "]";
```
Comment (khadri2021) [inline src/com/khadri/hibernate/entity/College.java:109]: use camelcase

File: src/college-mapping.xml (hunk 1 above)
Comment (Babjanshake) [reply src/college-mapping.xml:15]: It is camelCase only.

File: src/com/khadri/hibernate/client/Client2.java (hunk 2 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/client/Client2.java]: Changed Successfully.

File: src/com/khadri/hibernate/client/Client2.java (hunk 3 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/client/Client2.java]: It is camelCase only.

File: src/com/khadri/hibernate/client/Client2.java (hunk 4 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/client/Client2.java]: Changed Successfully.

File: src/com/khadri/hibernate/entity/College.java (hunk 5 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/entity/College.java:13]: It is camelCase only.

File: src/com/khadri/hibernate/entity/College.java (hunk 6 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/entity/College.java:109]: It is camelCase only.

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**